CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Version 1.1.1
```

Comment:
Can you please update also the compatibility table of the README.md with 1.1.x instead of 1.1.0

Reply:
Updated